src/platform/user/profile/vap-svc/containers/VAPServiceProfileField.jsx

```diff
@@ -187,21 +180,22 @@ class VAPServiceProfileField extends React.Component {
     }
     return (
       !this.props.isEmpty &&
-      (!transactionPending ||
-        this.props.isEditing ||
-        this.props.showValidationModal)
+      !(this.props.isEditing || this.props.showValidationModal) &&
+      !transactionPending
```

Comment:
Yep, now that the editing is done 'inline' via the ProfileInformationFIeldController, this logic changes a bit so that the edit link is only shown when the user is not actively editing, the address validation 'modal' ui is not being shown, and when a transaction/api call is not ongoing.